tests/test_wallet.py

```diff
@@ -0,0 +1,144 @@
+from decimal import Decimal
+from unittest import IsolatedAsyncioTestCase
+
+from eth_account import Account
+from web3 import HTTPProvider, Web3
+
+from zksync_sdk import ZkSyncLibrary
+from zksync_sdk.ethereum_provider import EthereumProvider
+from zksync_sdk.ethereum_signer import EthereumSignerWeb3
+from zksync_sdk.network import rinkeby
+from zksync_sdk.transport.http import HttpJsonRPCTransport
+from zksync_sdk.types import ChangePubKeyTypes, Token
+from zksync_sdk.wallet import Wallet
+from zksync_sdk.zksync import ZkSync
+from zksync_sdk.zksync_provider import ZkSyncProviderV01
+from zksync_sdk.zksync_provider.types import TransactionWithSignature
+from zksync_sdk.zksync_signer import ZkSyncSigner
```

Comment:
These imports look huge, almost Java-like. Maybe we should re-export essential package parts in the main `__init__.py`?

Reply:
This would allow convenient usage of namespaced classes and functions if we do something like `import zksync_sdk as zk`